docs/getting-started.md

```diff
@@ -199,7 +199,32 @@ You can view restore statues with `ListRestoreStatuses`.
 $ grpcurl -plaintext -d '{"limit": 3, "cluster_id": "CLUSTER-ID"}' 192.168.0.254:20051 rpc.Cassy.ListRestoreStatuses
 ```
 
-## Future Work
+### Scheduling backups
 
-The following features are planned to be implemented in the near future.
-* GUI component to make backup and restore operations even easier. 
+You can schedule backups using the `cassy-schedule` executable together with `cron`. 
+
+#### Environment variables
+First, you will need to set environment variables to tell the scheduler where your Cassy server is running. If you do not set these variables, the scheduler will default to `localhost:20051`.
+
+| Variable | Definition | Example
+| -------- | ---------- | ---------------------------------- |
+| CASSY_SCHEDULER_HOST  | the ip address of the cassy server | 127.0.0.1
+| CASSY_SCHEDULER_PORT  | the port of the cassy server       | 20051
+
+#### Usage
+
+The command can be executed with `cassy-schedule [-t=<timeout>] [subcommand] [options]`. The `timeout` option is an integer in seconds, which will set a maximum time to wait for the backup to reach a `COMPLETED` or `FAILED` status, or otherwise exit the program with an exception. If you do not set this option it will use the default value of 20.
```

Comment:
I think the default value 20 seconds is too short. Is there any particular reason for the number?

I think backups usually take hours in normal production systems, so it should be something like 18000 (5 hours).

Reply:
I had no idea how long the backups might potentially take, so 20 seconds is just an arbitrary default value. I will update it to 18000, and the user can change the time as desired using the `-t` flag.

I could also update the timer to use minutes/hours instead of seconds if that seems like the more typical usage.